CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
 
 ### Fixes
 - fixed image data-source bug when `name` not provided - data-source returned 0 results
+- immutable fields will throw error in plan phase, thus the state is no longer updated
```

Comment:
Here i move at the beginning what the bug is, because not everyone is aware of it, and it's easier to scan the changelog if it happens to have this bug